media_service/mediastore.py

```diff
@@ -24,6 +27,30 @@
 class MediaStoreUploadException(Exception):
     pass
 
+def process_zip(filelist):
+    '''
+    processes a file upload list, unzipping all zips
+    returns a new list with unzipped files
+    '''
+    newlist = []
+    for file in filelist:
+        if "zip" in file.content_type:
+            # unzip and append to the list
+            zip = zipfile.ZipFile(file, "r")
```

Comment:
It looks like there are two exceptions defined by the [zipfile](https://docs.python.org/2/library/zipfile.html) module:

1. zipfile.BadZipFile
2. zipfile.LargeZipFile

Should we be trying to catch either of those here? Or maybe they should be handled by the caller (in the view) instead?